modules/interpretability/covid_classification.ipynb

```diff
@@ -69,18 +117,10 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "id": "6330ae42",
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "/nfs/home/rbrown/Documents/Data/MONAI\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "directory = os.environ.get(\"MONAI_DATA_DIRECTORY\")\n",
```

Comment:
Can we add the import of `shutil` and these markdown and code cells since we create a temporary data folder? Thanks!
```
## Cleanup data directory
Remove directory if a temporary was used.
```
```
if directory is None:
    shutil.rmtree(root_dir)
```